### ISSUE TITLE
NET-1573 Fix S3900 FN: Rule does not raise on LINQ methods

### ISSUE THREAD

**Issue by Tim-Pohlmann** [SonarSource/sonar-dotnet#9500]
```csharp
void Method(List<int> list)
{
    _ = list.Where(x => true);  // FN
}
```
S3900 should re-use the list of known LINQ methods introduced in #9414 to raise in cases like this.

**Comment by pavel-mikula-sonarsource** [comment]
Internal ticket [NET-1573](https://sonarsource.atlassian.net/browse/NET-1573)

[NET-1573]: https://sonarsource.atlassian.net/browse/NET-1573?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by pavel-mikula-sonarsource** [comment]
Moved
